aggregation-worker/src/main/java/au/org/emii/aggregationworker/AggregationRunner.java

```diff
@@ -408,4 +539,15 @@ private String formatPeriodString(Period period) {
 
         return formatter.print(period);
     }
+
+
+    private String getMetadataFilename(String collectionTitle) {
+        if(collectionTitle == null || collectionTitle.trim().length() == 0) {
+            //  Use a default filename
+            return DEFAULT_METADATA_FILENAME;
+        }
+
+        //  Replace spaces with underscores
+        return collectionTitle.replace(" ", "_") + METADATA_FILE_EXTENSION;
```

Comment:
May be other unacceptable file name characters in a title?  Must be some standard code to use for that

Reply:
Replaced with regex replace.